src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -834,6 +836,9 @@ bool StorageReplicatedMergeTree::createTableIfNotExists(const StorageMetadataPtr
         ops.emplace_back(zkutil::makeCreateRequest(replica_path + "/mutation_pointer", "",
             zkutil::CreateMode::Persistent));
 
+        ops.emplace_back(zkutil::makeCreateRequest(replica_path + "/creator_uuid", toString(ServerUUID::get()),
+            zkutil::CreateMode::Persistent));
```

Comment:
Don't forget to update `createNewZooKeeperNodes` as well

Reply:
Will there be any other use for this node? Because otherwise, I don't see a reason for adding it to the `createNewZooKeeperNodes`. We call this function in the constructor only after we successfully initialized our replica nodes, and even if the case we are trying to solve happened on the older version and then we updated it we won't get to the `createNewZooKeeperNodes` and fail earlier.